proc_fusion_spell.lua

```diff
@@ -116,6 +116,35 @@ function(fusfilter,matfilter,extrafil,extraop,gc2,stage2,exactcount,value,locati
 					Fusion.CheckExact=exactcount
 					Fusion.CheckMin=mincount
 					Fusion.CheckMax=maxcount
+					--"Cyberdark Chimera" OPT EFFECT_EXTRA_FUSION_MATERIAL check
+					local extra_feff_mg=mg1:Filter(Card.IsHasEffect,nil,EFFECT_EXTRA_FUSION_MATERIAL)
+					if #extra_feff_mg>0 then
+						local extra_feff=extra_feff_mg:GetFirst():IsHasEffect(EFFECT_EXTRA_FUSION_MATERIAL)
```

Comment:
This might interfere with the current workaround in https://github.com/ProjectIgnis/CardScripts/blob/a1b117172a09e5ddd460ea2100bd09c1a5ee6da7/proc_fusion_spell.lua#L16-L25, as the cards will always have EFFECT_EXTRA_FUSION_MATERIAL registered to them, maybe better check for the effect not to be ``geff`` and in case get the next ``EFFECT_EXTRA_FUSION_MATERIAL``?